dv/uvm/icache/data/common_modes.hjson

```diff
@@ -0,0 +1,38 @@
+// Copyright lowRISC contributors.
```

Comment:
Consider moving these common hjson files to a more common area the Ibex testbench can point to as well.

Reply:
Yep, I'm thinking about how to structure the shared directories